README.md

```diff
@@ -83,32 +106,38 @@ git clone https://bitbucket.org/fathomteam/moab -b Version5.1.0
 cd moab
 autoreconf -fi
 cd ../bld
-../moab/configure --disable-blaslapack \
-                  --enable-shared \
+../moab/configure --enable-shared \
                   --enable-optimize \
                   --disable-debug \
                   --disable-blaslapack \
                   --with-eigen3=/usr/include/eigen3 \
                   --with-hdf5=/usr/lib/x86_64-linux-gnu/hdf5/serial \
-                  --prefix=${HOME}/plugin-build/moab
+                  --prefix=${HOME}/plugin-build/moab \
+                  CXXFLAGS=-D_GLIBCXX_USE_CXX11_ABI=0
```

Comment:
could you put the additional flag in the same place as in the other make commands ?

Reply:
I'm actually not sure if this is possible (it probably is) but I'll check

Reply:
I moved the ABI flag to the top